kolibri/plugins/learn/assets/src/views/TopicsPage/HeaderTabs.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+
+  <!-- Stolen from Coach -->
+  <div
```

Comment:
Does this component get reused at all? It feels like too much breaking out if not.

Reply:
Nah it's just the structure that came with it when I stole it from Coach - seems more reasonable to make something specific to the TopicsPage anyway here and would go hand-in-hand with the work to move them per @jtamiace 's latest TopicsPage updated designs